manual/manual/tutorials/moduleexamples.etex

```diff
@@ -85,8 +85,9 @@ Two constructions are available for this purpose:
 \end{caml_example}
 and
 \begin{caml_example}{toplevel}
-  PrioQueue.(insert empty 1 "hello");;
+  let PrioQueue.(Node _ | Empty) = PrioQueue.(insert empty 1 "hello");;
 \end{caml_example}
+Note that in patterns, only the second form is available.
```

Comment:
I don't find this change of the example, and the note, very readable. I think that it would be nicer to wait for the end of the paragraph (line 101/102) and say: "This dotted form is also available in pattern: {examples}".

Reply:
You are right, the example was trying to be too clever.